client/src/components/dashboard/Dashboard.js

```diff
@@ -1,15 +1,179 @@
-import Container from "react-bootstrap/Container"
+/**
+ * This file exports the inner content of Researchify Dashboard Page
+ */
+import React, { useState } from 'react'
+import { Container, Button, Modal, Image, CardGroup, Card, Form } from "react-bootstrap"
 
-import './Dashboard.css'
+/** Redux **/
+import { useDispatch, useSelector } from 'react-redux';
+import { createWebsite } from '../../actions/website';
+
+import { Link } from 'react-router-dom';
 
+/** icons **/
+import { BsPencilSquare, BsServer, BsDisplayFill, BsChevronRight, BsChevronLeft, BsCheck } from 'react-icons/bs'
+/** css **/
+import './Dashboard.css'
+// import DashboardModal from './DashboardModal'
+import homeTemplateImg1 from '../../images/home_page_template1.jpg'
+import homeTemplateImg2 from '../../images/home_page_template2.jpg'
 
 const Dashboard = () => {
+
+    const [show1, setShow1] = useState(false);
+    const [show2, setShow2] = useState(false);
+    const [show3, setShow3] = useState(false);
+
+    const handleCloseOne = () => setShow1(false);
+    const handleCloseTwo = () => setShow2(false);
+    const handleCloseThree = () => setShow3(false);
+
+    const handleShowOne = () => setShow1(true);
+    const handleShowTwo = () => setShow2(true);
+    const handleShowThree = () => setShow3(true);
+
+
+    const dispatch = useDispatch();
+    const websiteIsCreated = useSelector(state => state.website.isCreated);
+
     return (
-        <main>                
-            <Container fluid>
-                You are currently viewing dashboard page. 
-            </Container>
-        </main>
+
+        <Container fluid className="researchify-dashboard-container">
+            <Card className="text-center researchify-dashboard-card">
+
+                <Card.Body>

```

Comment:
I believe this is the body which contains the card in the darshboard, if so change the width to be adaptive and aligned to the center